applications/ConvectionDiffusionApplication/tests/cpp_tests/test_eulerian_diff.cpp

```diff
@@ -163,8 +163,18 @@ namespace Testing
         expected_LHS(3, 2) = 0.0833333;
         expected_LHS(3, 3) = 0.25;
 
+        // Test CalculateLocalSystem
+        auto p_element = r_test_model_part.pGetElement(1);
+        Vector RHS = ZeroVector(4);
+        Matrix LHS = ZeroMatrix(4,4);
+        p_element->CalculateLocalSystem(LHS, RHS, r_test_model_part.GetProcessInfo());
         KRATOS_CHECK_VECTOR_NEAR(RHS, expected_RHS, 1.0e-4)
```

Comment:
why so few decimals?

Reply:
This is what we used to have. I changed it to 10^-12 in all the tests.